tools/proteomiqon_peptidedb/proteomiqon_peptidedb.xml

```diff
@@ -0,0 +1,206 @@
+        <test expect_num_outputs="2">
+            <param name="fasta" value="sample.fasta"/>
+            <param name="Name" value="test"/>
+            <param name="ParseProteinIDRegexPattern" value="id"/>
+            <param name="Protease" value="Trypsin"/>
+            <param name="MinMissedCleavages" value="0"/>
+            <param name="MaxMissedCleavages" value="2"/>
+            <param name="MaxMass" value="15000.0"/>
+            <param name="MinPepLength" value="4"/>
+            <param name="MaxPepLength" value="65"/>
+            <param name="IsotopicMod" value="N15"/>
+            <param name="MassMode" value="Monoisotopic"/>
+            <param name="FixedMods" value="Acetylation'ProtNTerm',Carbamidomethyl'Cys',Oxidation'Met',Phosphorylation'Ser'Thr'Tyr',Pyro_Glu'GluNterm',Pyro_Glu'GlnNterm'"/>
+            <param name="VariableMods" value="Acetylation'ProtNTerm',Carbamidomethyl'Cys',Oxidation'Met',Phosphorylation'Ser'Thr'Tyr',Pyro_Glu'GluNterm',Pyro_Glu'GlnNterm'"/>
+            <param name="VarModThreshold" value="4"/>
+            <param name="outputParamfile" value="true"/>
+            <output name="out_paramfile" file="result_2.json"/>
+        </test>
+        <test expect_num_outputs="2">
+            <param name="fasta" value="sample.fasta"/>
+            <param name="Name" value="test"/>
+            <param name="ParseProteinIDRegexPattern" value="id"/>
+            <param name="Protease" value="Trypsin"/>
+            <param name="MinMissedCleavages" value="0"/>
+            <param name="MaxMissedCleavages" value="2"/>
+            <param name="MaxMass" value="15000.0"/>
+            <param name="MinPepLength" value="4"/>
+            <param name="MaxPepLength" value="65"/>
+            <param name="IsotopicMod" value=""/>
+            <param name="MassMode" value="Average"/>
+            <param name="FixedMods" value=""/>
+            <param name="VariableMods" value=""/>
+            <param name="VarModThreshold" value="4"/>
+            <param name="outputParamfile" value="true"/>
+            <output name="out_paramfile" file="result_3.json"/>
+        </test>
+    </tests>
+    <help>
+    <![CDATA[
+Introduction
+------------
+MS-based shotgun proteomics estimates protein abundances using a proxy: peptides. 
+An established method to identify acquired MS/MS spectra is the comparison of each spectrum with peptides in a reference database. 
+
+What It Does
+------------
+The PeptideDB tool helps to create peptide databases by in silico digestion given proteome information in the FASTA format and a set of parameters 
+that allow the user to mimic conditions of their specific experiment. 
+The created database stores peptide protein relationships in a SQLite database which can then be supplied to other ProteomIQon tools.
+
+Further Reading
+---------------
+Additional information about the tool can be found in the `documentation <https://csbiology.github.io/ProteomIQon/tools/PeptideDB.html>`_.
+    ]]>
+    </help>
```

Comment:
add ciations?

Reply:
Is in preparation and will be added upon publication.